packages/material-ui/src/Slider/Slider.js

```diff
@@ -1,16 +1,13 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { chainPropTypes } from '@material-ui/utils';
-import SliderUnstyled, {
-  SliderValueLabelUnstyled,
-  sliderClasses,
-} from '@material-ui/unstyled/SliderUnstyled';
+import { SliderUnstyled, SliderValueLabelUnstyled, sliderClasses } from '@material-ui/unstyled';
```

Comment:
The motivation was to allow:

```jsx
import Slider from '@material-ui/core/Slider';
```

to be an effective way to tree-shake the bundle, even in development. If we move in this direction, I think that we might as well update the demos. We would load the whole unstyled package. it seems to yield the same properties with a nicer DX:

```jsx
import { Slider } from '@material-ui/core';
```

I think that we should be careful with how it will scale. We can look at the icon for that.
Next.js renders the following page in less than 1s ✅.

```jsx
import React from 'react';
import CloseIcon from '@material-ui/icons/Close';

export default function LandingPage() {
  return (
    <div>
      <CloseIcon />
    </div>
  );
}
```

However, If you take this demo, it takes **~60s** for Next.js to render it ❌.

```jsx
import React from 'react';
import { Close as CloseIcon } from '@material-ui/icons';

export default function LandingPage() {
  return (
    <div>
      <CloseIcon />
    </div>
  );
}
```

Reply:
Let's focus on the concrete case not icons.

Reply:
Oh yeah, icon was probably not the best comparison source I could have picked.

```jsx
import Alert from '@material-ui/core/Alert';
```

vs.

```jsx
import { Alert } from '@material-ui/core';
```

would have been better considering that unstyled will have about the same amount of modules as core. It turns out that there isn't much difference in this Alert case. I guess I was more concerned about what would happen if we x2 the number of components in the core. Would we start to hit a slowness?

Reply:
@eps1lon Is this measurement better 🔼 :). Do you agree with the hypothesis that below 10s we are good and above it starts to be an issue?